setup.py

```diff
@@ -50,6 +50,7 @@
     install_requires=[
         'numpy>=1.11',
         'scipy>=1.0.0',
+        'pybloomfiltermmap3>=0.6.0',
```

Comment:
Let's make this optional by using extras. And indicate in the API doc the installation instructions.